esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -468,7 +468,7 @@ bool CWsESDLConfigEx::onPublishESDLBinding(IEspContext &context, IEspPublishESDL
             resp.updateStatus().setCode(m_esdlStore->bindService(espBindingName.str(),
                                                            methodstree.get(),
                                                            espProcName.str(),
-                                                           espPort,
+                                                           espPort.str(),
```

Comment:
I don't believe this change will have any impact - the operator char*() function is the same as calling str().

What symptom had been observed that you thought this would fix?

Reply:
@richardkchapman the symptom is that we saw one binding that has seemingly garbled data as the port field. We now believe the garbled port likely came from the caller of the web service,  intentionally or due to un-initialized buffer. The newly added port validation will prevent the binding from being added with an invalid port, so this is still a valid fix.